components/FilterButtons.vue

```diff
@@ -1,17 +1,34 @@
 <template>
-    <div>
-        <button>{{ $t("All") }}</button>
-        <button>{{ $t("Incomplete") }}</button>
-        <button>{{ $t("Complete") }}</button>
-    </div>
+  <div>
+    <button @click="showAll" :class="{ 'is-active': currentFilter === 'All' }">{{ $t("All") }}</button>
```

Comment:
We also follow some convention while writing i18n keys. As **All** is a general text that can be used in many places, we'll often want to prefix it with `general`. 

So it'll be, `general.all` or `general.cta.all` where cta means call to action. 
```suggestion
    <button @click="showAll" :class="{ 'is-active': currentFilter === 'All' }">{{ $t("general.all") }}</button>
```